toolparameteriser/run.py

```diff
@@ -5,42 +5,44 @@
 import tomllib
 import logging,os
 import argparse
-toolparameteriser.utils.setlogging()
 
 def main(args=None):
 
     if not args:
         parser = argparse.ArgumentParser(description='Run/analyse a tool test')
         parser.add_argument('-c','--config_path', metavar='path', required=True,
                             help='the path to configuration file')
-        parser.add_argument('-D','--dryrun', metavar="bool", required=False,
-                           help='if true jobs will not run')
+        parser.add_argument('-D','--dryrun', required=False, default = False, action = 'store_true', 
```

Comment:
`required=False` and `default=False` are defaults for `action="store_true": https://docs.python.org/3/library/argparse.html#action
```suggestion
        parser.add_argument('-D','--dryrun', action = 'store_true', 
```